Add description for GitHub's auto-merge feature (#283)
otterize/docs: docs/features/github/tutorials/automated-pull-requests.mdx

5. In the `Open PR on` section, select the `Repository` field, and provide the owner and organization names in the form: *<github user name\>/otterize-tutorial-github*.
6. Select the `Base Branch` field, and enter *main*.
7. Select the `ClientIntents path` field, and enter `intents`. It represents the relative path hosting the ClientIntents manifests.

> 8. You can set the pull request to be auto-merged, using GitHub's merge queue, by enabling the auto merge toggle.

Reviewer: ```suggestion
8. You can set the pull request to be auto-merged, by enabling the auto merge toggle.
```

I think this doesn't actually use merge queue, which is a separate GitHub Enterprise feature that queues up PRs to be merged, including rebasing each one in line to include all the others, so that PRs can go in quickly and have all previous code as part of their tests, to avoid cross-breakage (the case when all PRs pass tests, but when combined in main they break)

>    Note: for this to work, you have to make sure that you enable `auto-merge` feature in your GitHub repository. For more information about GitHub's auto merge, follow [GitHub's instruction](https://docs.github.com/en/repositories/configuring-branches-and-merges-in-your-repository/configuring-pull-request-merges/managing-auto-merge-for-pull-requests-in-your-repository#managing-auto-merge).

Reviewer: ```suggestion
   Note: for this to work, you have to make sure that you enable the `auto-merge` feature for the GitHub repository. For more information about GitHub's auto merge, follow [GitHub's instructions](https://docs.github.com/en/repositories/configuring-branches-and-merges-in-your-repository/configuring-pull-request-merges/managing-auto-merge-for-pull-requests-in-your-repository#managing-auto-merge).
```
